### DISCUSSION TITLE
Idioma de tradução Brasileiro sem .json no final e não aceita instalação VERSÃO 9.0.6.1008 (64-bit)

### DISCUSSION THREAD

**Discussion by fapj** [d2phap/ImageGlass#1678] [General]
O mesmo foi instalado com inglês, mas não instala o PT-BR baixado original, no início pensei que era porque estava vindo o arquivo sem JSON no final, mas mesmo colocando e ele reconhecendo-o não instala dando erro.

A pergunta é a seguinte, seria possível ter os Downloads do software traduzidos ao invés de baixar e instalar a tradução, ou baixar o software já com a escolha de baixar as traduções escolhidas junto ao aplicativo, isso mostraria uma melhora significativa pra cada pessoa de sua linguagem origem, para se identificar melhor e não ter problemas deste tipo?

![Image glass tradução bugada](https://github.com/d2phap/ImageGlass/assets/67239968/c3ca25ca-dd17-4031-adf6-8ae28707268f)

**Comment by d2phap** [comment]
All language packs are not compatible with v9.0. Please wait for the official release because I need to ask help from the people to translate ImageGlass v9.0

**Comment by fapj** [comment]
OK vou aguardar.

Em qua., 18 de out. de 2023 às 23:11, Phap Dieu Duong <
***@***.***> escreveu:
